tests/functional/base_test.go

```diff
@@ -78,6 +78,22 @@ func CreateTransportURLSecret(name types.NamespacedName) *corev1.Secret {
 	return secret
 }
 
+func CreateTransportURLSecretWithQuorumQueues(name types.NamespacedName, quorumQueues bool) *corev1.Secret {
```

Comment:
@lmiccini should this be updated to point to the function provided by infra-operator?